modules/CoreLibrary/Sources/Services/Delivery/DeliveryClient.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import ServicesInterface
+import Networking
+
+public struct DeliveryClient: DeliveryClientProtocol {
+    private let networkManager: NetworkManagerProtocol
+    
+    public init(networkManager: NetworkManagerProtocol = NetworkManager()) {
```

Comment:
Let's work on DI approaches today to avoid this `networkManager: NetworkManagerProtocol = NetworkManager()` and make it more scalable.